test/test_um2netcdf.py

```diff
@@ -683,21 +683,31 @@ def level_heights():
     #     fix_level_coords() only accesses height array[0]
     return [20.0003377]
 
+
 @pytest.fixture
 def level_coords(level_heights):
     return {um2nc.MODEL_LEVEL_NUM: iris.coords.DimCoord(range(1, 39)),
             um2nc.LEVEL_HEIGHT: iris.coords.DimCoord(level_heights),
             um2nc.SIGMA: iris.coords.AuxCoord(np.array([0.99771646]))}
 
+
```

Comment:
The fixture below should be replaceable with the `DummyCubeWithCoords` from the lat/long fixes.

Reply:
Issue #63 covers this task.